Lib/pdb.py

```diff
@@ -709,6 +709,27 @@ def _get_asyncio_task(self):
             task = None
         return task
 
+    def _get_pid_from_process(self, process):
+        """process could be a subprocess.Popen, multiprocessing.Process or a pid
+        """
+        # They are not used elsewhere so do a lazy import
+        from multiprocessing import Process
+        from subprocess import Popen
+
+        try:
+            process = self._getval(process)
+        except:
+            # Error message is already displayed
+            return None
+
+        if isinstance(process, (Process, Popen)):
+            return process.pid
+        elif isinstance(process, int):
+            return process
+
+        self.error(f"Invalid process {process}")
```

Comment:
Probably should use the repr here, instead:
```suggestion
        self.error(f"Invalid process {process!r}")
```